services/webnn/ort/graph_builder_ort.cc

```diff
@@ -1225,7 +1227,8 @@ GraphBuilderOrt::AddDequantizeLinearOperation(
     // Currently, OpenVINO only supports axis == 0 when scale.size == 2.
     // https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/frontend/src/op/dequantize_linear.cpp#L228.
     if (base::CommandLine::ForCurrentProcess()->HasSwitch(
-            switches::kWebNNOrtUseOpenvino)) {
+            switches::kWebNNOrtUseOpenvino) &&
+        std::is_same_v<DequantizeOrQuantizeLinear, mojom::DequantizeLinear>) {
       if (scale_shape.size() != 2) {
         // https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/frontend/src/op/dequantize_linear.cpp#L220
```

Comment:
The pointer is only for dequantizeLinear. Is the quantizeLinear  the same? Could you please add a pointer?

Reply:
From this file- https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/onnx/frontend/src/op/quantize_linear.cpp, I can't find any restricts for block-wise quantizeLinear. And it seems only validate opset13 quantizeLinear which has not supported block_wise. But it indeed pass block_wise quantizeLinear WPT tests for gpu devicetype.

Reply:
If the 2D scale restriction doesn't apply to quantizeLinear op, you may want to exclude it?

Reply:
I added `&&std::is_same_v<DequantizeOrQuantizeLinear, mojom::DequantizeLinear>` condition to this code snippets. I suppoose it can exclude quantizeLinear.  Am I right?